src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,9 +1,18 @@
 package mate.academy
 
-fun greetUsers() {
-    // implement function
-}
-
 fun main() {
     greetUsers()
 }
+
+fun greetUsers() {
+    var flag = true
+    while (flag) {
+        println("Please enter a name (or press Enter to quit): ")
+        val name : String? = readLine()
```

Comment:
In Kotlin it is allowed not to write variable's type directly. This is not mistake, just FYI